src/gfx/reverse.cpp

```diff
@@ -241,24 +242,27 @@ void reverse() {
 
 	if (tilemap) {
 		if (attrmap) {
-			for (auto [id, attr] : zip(*tilemap, *attrmap)) {
-				bool bank = attr & 1 << 3;
-				if (id >= options.maxNbTiles[bank]) {
+			for (auto [tileID, attribute] : zip(*tilemap, *attrmap)) {
+				if (bool bank = attribute & 1 << 3; tileID >= options.maxNbTiles[bank]) {
 					warning(
-					    "Tile #%" PRIu8 " was referenced, but the limit for bank %u is %" PRIu16,
-					    id,
+					    "Tilemap references tile #%" PRIu8 ", but the limit for bank %u is %" PRIu16,
```

Comment:
It would be nice if that error message could also mention the tile's coordinates.

(And, for that matter, I think the attrmap and palmap messages could also make use of that!)